resdk/resources/base.py

```diff
@@ -235,6 +235,29 @@ def permissions(self):
 
         return self._permissions
 
+    def _list_holders_with_perm(self, perm):
+        """Get users/group/public names that have ``perm`` perm."""
+        holders = []
+        self.permissions.fetch()
+
+        for item in self.permissions._permissions:  # pylint: disable=protected-access
+            if perm in item['permissions']:
+                holders.append(item.get('name', 'Public'))
+
+        return holders
+
+    def list_owners(self):
+        """List owners."""
```

Comment:
Make the docstring more informative, e.g. `Return list of users with ``owner`` permission.`